health-sdk/sdk/src/main/java/net/gini/android/health/sdk/integratedFlow/PaymentFragment.kt

```diff
@@ -82,7 +82,14 @@ data class PaymentFlowConfiguration(
      *
      * Note: Only taken into consideration for payments initiated with [documentId]
      */
-    val showCloseButtonOnReviewFragment: Boolean = false
+    val showCloseButtonOnReviewFragment: Boolean = false,
+    /**
+     * Defines the duration (in seconds) for which the payment review popup will be displayed before automatically closing.
+     * This value should be within a reasonable range (e.g., 0-10 seconds) to ensure a smooth user experience.
```

Comment:
[Request] Because we coerce the value to be between `0` and `10`, please specify this as a must, rather than a suggestion.